code change for translation

pengyin-shan opened the pull request:
Fixes #137  .

Changes proposed in this PR:

Some code changes for translations
- Click below to give data management staff at...
- API Access
- Allows the user to grant API access to organizations: translated
- Grant API access: translated

lagoan commented on app/views/usage/filter.js.erb:
```diff
@@ -1,5 +1,9 @@
-document.getElementById('filter_ranged').innerHTML = '<%= escape_javascript(render partial: "filtered_value", locals: { label: "New #{@topic}", value: @ranged.to_i }) %>';
-document.getElementById('filter_total').innerHTML = '<%= escape_javascript(render partial: "filtered_value", locals: { label: "Total #{@topic}", value: @total.to_i }) %>';
\ No newline at end of file
+document.getElementById('filter_ranged').innerHTML = '<%= escape_javascript(render partial: "filtered_value", 
+                                                    locals: { label: _("New %{topic}") % {topic: @topic }, 
+                                                    value: @ranged.to_i }) %>';
+document.getElementById('filter_total').innerHTML = '<%= escape_javascript(render partial: "filtered_value", 
+                                                    locals: { label: _("Total %{topic}") % {topic: @topic }, 
+                                                    value: @total.to_i }) %>';
```
I too get the urge to split long lines of code this way. However, most of ruby's syntax calls for longer lines of code. For now we should leave them in a single line, compare to the rest of the code base, and set rubocop rules to enforce them.

The newer version of roadmap does have a functioning rubocop workflow so we can address  these issues then.

lagoan reviewed (changes requested):
Things look good. The only changes I see would be of some lines that were split up and we should add an entry on the CHANGELOG file.

pengyin-shan commented:
> Things look good. The only changes I see would be of some lines that were split up and we should add an entry on the CHANGELOG file.

lol I always forgot the changelog part. Added it and changed the code to one-line style
